syntax = "proto3";
package test;

message Item{
  int32 id = 1;
}

message PBStruct{
  string str = 1;
  int32  id = 2;
  repeated int64 ids_array = 3;
  repeated string strs_array = 4;
  repeated Item item_array = 5;
  map<string, Item> item_map = 6;
  map<string, int32> str_int_map = 7;
}